### ISSUE TITLE
Crash on opening any non-player inventory (v3.3.0)

### ISSUE THREAD

**Issue by CharliePearce** [b0bst3r/Superstitions#26]
Reproduction steps:

1. Start new world
2. Place/Find chest in world
3. Open chest

Stacktrace/Log:
 ```
A detailed walkthrough of the error, its code path and all known details is as follows:
 ---------------------------------------------------------------------------------------
 
 -- Head --
 Thread: Client thread
 Stacktrace:
 	at elec332.core.main.ElecCore$1.onItemRightClick(ElecCore.java:164)
 	at net.minecraftforge.fml.common.eventhandler.ASMEventHandler_1209__onItemRightClick_RightClickBlock.invoke(.dynamic)
 	at net.minecraftforge.fml.common.eventhandler.ASMEventHandler.invoke(ASMEventHandler.java:90)
 	at net.minecraftforge.fml.common.eventhandler.EventBus.post(EventBus.java:185)
 	at net.minecraftforge.common.ForgeHooks.onRightClickBlock(ForgeHooks.java:1026)
 	at net.minecraft.client.multiplayer.PlayerControllerMP.func_187099_a(PlayerControllerMP.java:387)
 	at net.minecraft.client.Minecraft.func_147121_ag(Minecraft.java:1518)
 	at net.minecraft.client.Minecraft.func_184117_aA(Minecraft.java:2184)
 	at net.minecraft.client.Minecraft.func_184118_az(Minecraft.java:1961)
 
 -- Affected level --
 Details:
 	Level name: MpServer

 	Chunk stats: MultiplayerChunkCache: 560, 560
 	Level seed: 0
 	Level generator: ID 06 - quark_realistic, ver 0. Features enabled: false
 	Level generator options: 
 	Level spawn location: World: (92,64,248), Chunk: (at 12,4,8 in 5,15; contains blocks 80,0,240 to 95,255,255), Region: (0,0; contains chunks 0,0 to 31,31, blocks 0,0,0 to 511,255,511)
 	Level time: 2448 game time, 2448 day time
 	Level dimension: 0
 	Level storage version: 0x00000 - Unknown?
 	Level weather: Rain time: 0 (now: false), thunder time: 0 (now: false)
 	Level game mode: Game mode: survival (ID 0). Hardcore: false. Cheats: false


 	Server brand: fml,forge
 	Server type: Integrated singleplayer server
 Stacktrace:
 	at net.minecraft.client.multiplayer.WorldClient.func_72914_a(WorldClient.java:415)
 	at net.minecraft.client.Minecraft.func_71396_d(Minecraft.java:2660)
 	at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:400)
 	at net.minecraft.client.main.Main.main(SourceFile:124)
 	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
 	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
 	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
 	at java.lang.reflect.Method.invoke(Unknown Source)
 	at net.minecraft.launchwrapper.Launch.launch(Launch.java:135)
 	at net.minecraft.launchwrapper.Launch.main(Launch.java:28)
 
 -- System Details --
 Details:
 	Minecraft Version: 1.10.2
 	Operating System: Windows 10 (amd64) version 10.0
 	Java Version: 1.8.0_111, Oracle Corporation
 	Java VM Version: Java HotSpot(TM) 64-Bit Server VM (mixed mode), Oracle Corporation
 	Memory: 2105304720 bytes (2007 MB) / 4225236992 bytes (4029 MB) up to 4225236992 bytes (4029 MB)
 	JVM Flags: 8 total; -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -XX:-OmitStackTraceInFastThrow -Xms256M -Xmx4096M -XX:MetaspaceSize=256M -XX:+UseConcMarkSweepGC -XX:+CMSIncrementalMode -XX:-UseAdaptiveSizePolicy
 	IntCache: cache: 0, tcache: 0, allocated: 13, tallocated: 95
 	FML: MCP 9.32 Powered by Forge 12.18.3.2239 134 mods loaded, 133 mods active
 	States: 'U' = Unloaded 'L' = Loaded 'C' = Constructed 'H' = Pre-initialized 'I' = Initialized 'J' = Post-initialized 'A' = Available 'D' = Disabled 'E' = Errored
 (minecraft.jar) 
 (forge-1.10.2-12.18.3.2239-universal.jar) 
 (forge-1.10.2-12.18.3.2239-universal.jar) 
 (minecraft.jar) 
```

**Comment by b0bst3r** [comment]
Should be fixed in 3.3.1

**Comment by CharliePearce** [comment]
Tested, no longer getting this crash.
